addOns/help/src/main/javahelp/contents/start/proxies.html

```diff
@@ -113,6 +113,18 @@ <H3>OS X System Proxy settings</H3>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Press the</td><td> Network Options 'Apply' button</td></tr>
 </table>
 
+<H3>Linux Gnome Proxy settings</H3>
+
+<table>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Select the</td><td> 'Gnome Settings'</td></tr>
```

Comment:
from?

Reply:
The settings can be found from several places. I added the one from the corner since this one should not change.